### ISSUE TITLE
[Bug] Cannot attack

### ISSUE THREAD

**Issue by LeKopic** [kypvalanx/Foundry-VTT-StarWars-SagaEdition#512]
Buп: Cannot attack (the attack isn't rolled and doesn't appear in the chat).
Console: 
![image](https://github.com/user-attachments/assets/66dfa85b-1bf8-4680-867b-e81d5e9fcc76)
or
![image](https://github.com/user-attachments/assets/e21941c8-122f-435e-9425-eba39a5a16ab)

How to reproduce:
Add a new actor (in my case, a stormtrooper), select the token, target another token (in my case, spaarti stormtrooper), make an attack.

**Comment by Sajuuk2033** [comment]
I applied this fix myself to test before running a game. After applying the fix my actors roll for attacks again, but using the buttons to apply damage threw another error.

![image_2024-12-08_151133380](https://github.com/user-attachments/assets/47a33be3-f9c9-441e-8c34-e79ce42c12c0)

It appears line 232 under swse.mjs also needs to be updated in addition.
`function getHitOptionHTML(target, attack, tokenId) {
    let hit = target.system.defense.reflex.total <= attack;
`
I changed to 
`function getHitOptionHTML(target, attack, tokenId) {
    let hit = target.defense.reflex.total <= attack;
`
After applying this change the dialog to resolve attacks appears like so:
![image_2024-12-08_151546682](https://github.com/user-attachments/assets/fb3a5aaa-b956-4b23-9842-3ce8cc5610f0)

This lead to me consistently seeing an error about reading properties of undefined (reading  'damageThreshold') when targeting more than one actor to apply damage.  

![image_2024-12-08_153145306](https://github.com/user-attachments/assets/fcc49600-3d4e-4f55-8c93-edfe4cde43a9)
![image_2024-12-08_153158113](https://github.com/user-attachments/assets/3ae64b4d-d658-42dd-b211-209a8ea54f5a)


I have seen the same error when targeting a single actor, but I was testing in haste and did not manage to recreate that.
